tests/integration/test_convolve.py

```diff
@@ -82,6 +88,39 @@ def test_int(shape, filter_shape):
     check_convolve(a, v)
 
 
+def test_dtype():
+    shape = (5,) * 2
+    arr1 = num.random.randint(0, 5, shape)
+    arr2 = num.random.random(shape)
+    out_num = num.convolve(arr1, arr2, mode="same")
+    out_np = np.convolve(arr1, arr2, mode="same")
+    assert allclose(out_num, out_np)
+
+
+@pytest.mark.xfail
+@pytest.mark.parametrize("mode", MODES)
+def test_modes(mode):
+    shape = (5,) * 2
+    arr1 = num.random.random(shape)
+    arr2 = num.random.random(shape)
+    out_num = num.convolve(arr1, arr2, mode=mode)
+    # when mode!="same", cunumeric raises
+    # NotImplementedError: Need to implement other convolution modes
```

Comment:
But `same` does work? In that case we should have a working test intended for that case (either separate tests, or handled through pytest params)

Reply:
I would expect this exact same test to get run (without xfail) for the working case somehow